x-pack/plugins/security_solution/public/app/home/template_wrapper/index.tsx

```diff
@@ -39,6 +38,10 @@ const StyledKibanaPageTemplate = styled(KibanaPageTemplate)<
     $addBottomPadding?: boolean;
   }
 >`
+  ${({ $isShowingTimelineOverlay }) =>
+    $isShowingTimelineOverlay // When the timeline is open, the content underneath should have a fixed height to hide the scroll bar Y
+      ? `overflow: hidden; height: calc(100vh - ${HeaderHeight}px - ${TimelineBarHeight}px);`
+      : ''}
```

Comment:
This was suppose to hide the scroll bar Y when timeline is open. 

<img width="2552" alt="Screenshot 2023-06-21 at 11 51 09" src="https://github.com/elastic/kibana/assets/6295984/eaf0fe44-0b47-4f81-81cf-431175166c55">

However this does not fix it properly, I'll revert the change and follow up in a separate issue